composer.json

```diff
@@ -40,8 +40,8 @@
     },
     "require-dev": {
         "json-schema/JSON-Schema-Test-Suite": "1.1.0",
-        "phpunit/phpunit": "~3.7",
-        "phpdocumentor/phpdocumentor": "~2"
+        "phpdocumentor/phpdocumentor": "~2",
+        "phpunit/phpunit": "^4.7.7"
```

Comment:
@GrahamCampbell 

It probably is - however, I've made it a habit when updating packages to use the caret operator, and furthermore, specify the most recent version. It doesn't hurt, and the eventual package installed will be the same.